src/library/scala/collection/immutable/MapLike.scala

```diff
@@ -57,7 +57,7 @@ self =>
 
   protected[this] override def parCombiner = ParMap.newCombiner[K, V]
 
-  /** A new immutable map containing updating this map with a given key/value mapping.
+  /** A new immutable map updating this map with a given key/value mapping.
```

Comment:
There are no small typos, only small typists. 2.13 says
```
Creates a new map obtained by updating this map with a given key/value pair.
```
where `s/containing/obtained by/` minimizes the edit distance. Probably `a new map` is the key bit, so to speak.

In the series contain, obtain, maintain, what is a "stain"?